woocommerce/woocommerce-android
# AI: Some design tweaks and feature flag removal

Closes: #9446 


### Description
Design changes:
1. Updates AI button color on Aztec editor on dark mode
2. Make AI description promo dialog's title font smaller in size.

Code changes:
1. Removes feature flags related to AI sharing and AI description.


### Testing instructions


1. Set app in dark mode, go to a product details, tap Description, ensure the AI button on the left has a background color that matches the rest of the buttons.

![Screenshot 2023-08-29 at 14 34 16](https://github.com/woocommerce/woocommerce-android/assets/266376/dca38fc5-3e25-452b-ab20-a4dd5d0c7985)


2. Go to MyStoreViewModel, find these lines:
```
        if (isAIProductDescriptionEnabled() && !appPrefsWrapper.wasAIProductDescriptionPromoDialogShown) {
            triggerEvent(ShowAIProductDescriptionDialog)
            appPrefsWrapper.wasAIProductDescriptionPromoDialogShown = true
        }
```

and comment out the first and last line above. Run the app and the dialog will appear on My Store. Compare the shown font size with the figma source:


![Screenshot 2023-07-19 at 17 06 37](https://github.com/woocommerce/woocommerce-android/assets/2663464/50e75707-e419-4606-a3ba-44e7cae738d0)


Note that font size might differ also depending on phone's font setting.                                                                                                                                            


3. For the feature flags removal, test by making sure that both AI sharing and AI description generation still work.
